### PR TITLE
Loader animation

### LINKED ISSUE #56: [Add] Loading animation
Displaying an image can take up to 30 seconds. The user should let be known that the app processes.

### PR DESCRIPTION
**Pull request by maraPislar**
I added a loader animation so the user knows that by pressing "Generate" something happends in the backend.

This is a preview, please tell me what you think about it:

image

### CODE REVIEW

File: src/main/webapp/styles/style.css [hunk 1]
```diff
@@ -261,3 +260,28 @@ body {
...
 }
+
+.loader {
+  animation: spin 2s linear infinite;
+  border: 16px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 10px solid blue;
+  border-right: 10px solid red;
+  border-bottom: 10px solid yellow;
+  border-left: 10px solid green;
```
Comment (Krechals) [inline src/main/webapp/styles/style.css]: These declarations need to be placed into alphabetical order.

File: README.MD [hunk 2]
```diff
@@ -1 +0,0 @@
-# GPostCard
```
Comment (Krechals) [inline README.MD:1]: Make sure to not delete your previous Pull Request (README.MD).

**Review by Krechals — APPROVED** [review]

File: README.MD (hunk 2 above)
Comment (maraPislar) [reply README.MD:1]: Everything is fine about that.

File: src/main/webapp/styles/style.css [hunk 3]
```diff
@@ -261,3 +260,28 @@ body {
     transition: font-size 0.3s ease-in-out;
   }
 }
+
+.loader {
+  animation: spin 2s linear infinite;
+  border: 16px solid #f3f3f3;
```
Comment (jeadorf) [inline src/main/webapp/styles/style.css]: The `border` attribute seems redundant, given that you set them individually for each of the four directions.

File: src/main/webapp/styles/style.css [hunk 4]
```diff
@@ -261,3 +260,28 @@ body {
...
+  border-right: 10px solid red;
+  border-top: 10px solid blue;
+  height: 50px;
+  margin-left: 420px;
+  margin-top: 20px;
+  -webkit-animation: spin 2s linear infinite;
+  width: 50px;
+}
+
+@-webkit-keyframes spin {
```
Comment (jeadorf) [inline src/main/webapp/styles/style.css:278]: I'm curious to see this in action (it's beyond my CSS knowledge which is a few years behind).

**Review by jeadorf — APPROVED** [review]

File: src/main/webapp/styles/style.css (hunk 4 above)
Comment (maraPislar) [reply src/main/webapp/styles/style.css:278]: Sure! I will deploy so you can see it as well.

**Pull request merged**